app/controllers/users_controller.rb

```diff
@@ -252,6 +252,24 @@ def handle_settings_tab
     end
   end
 
+  def update_password
+    set_current_tab("account")
+
+    if @user.update_with_password(password_params)
```

Comment:
Most of this `if` is a copy/paste from `update`. I'd like to eventually refactor this whole controller, but until then this should do...